src/Popover/Popover.js

```diff
@@ -10,7 +10,6 @@ import Card from '../Card';
 
 const PopoverContent = styled(Card)`
   background: ${props => props.theme.colors.white};
-  border-radius: ${props => props.theme.radii.large};
```

Comment:
Just a heads up, I moved PopoverContent into its own file in my datepicker PR.

Reply:
Maybe you wanted them next to each other since the differences is so small?

Reply:
It could probably be done in separate stories, I just ended up doing so to match the figma design which has them together.